pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -604,3 +606,21 @@ func (cr *PerconaServerMongoDB) StatefulsetNamespacedName(rsName string) types.N
 func (cr *PerconaServerMongoDB) MongosNamespacedName() types.NamespacedName {
 	return types.NamespacedName{Name: cr.Name + "-" + "mongos", Namespace: cr.Namespace}
 }
+
+func (cr *PerconaServerMongoDB) CanBackup() error {
+	if cr.Status.State == AppStateReady {
+		return nil
+	}
+
+	if !cr.Spec.UnsafeConf {
```

Comment:
@delgod @hors 
Should we do this?
In that case, user will have to set this value to true, which could cause other issues with cluster

Reply:
It's based on the comments in the issue: https://jira.percona.com/browse/K8SPSMDB-365?focusedCommentId=278051&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-278051

Reply:
this thing is escalated by Support, please read the initial issue